.github/workflows/QAConfigMyApp.yml

```diff
@@ -63,6 +63,52 @@ jobs:
           curl https://gist.githubusercontent.com/iogbole/48e7568454b066132700c4fe039c2cff/raw/4aa417193e7ce9f3cce2410e67d525761cb6d678/gistfile1.txt -o ./custom_dashboards/CustomDashboard_vanilla.json
           export CMA_UPLOAD_CUSTOM_DASHBOARD=true
           ./start.sh -a IoT_API -c configmyappdemo-2044no-uzyczrm0.appd-cx.com -p appd -u appd
+
+    - name: TestCase7- health rules only - no overwrite - parameters
+      env:
+        CMA_USE_HTTPS: false
+      run: |
+          echo Running health rules only, get values from runtime parameters,
+          ./start.sh -a Jenkins_API -c configmyappdemo-2044no-uzyczrm0.appd-cx.com -p appd -u appd --health-rules-only --no-health-rules-overwrite
+
+    - name: TestCase8- health rules only - overwrite - env variables
+      env:
+        CMA_USE_HTTPS: false
+        CMA_HEALTH_RULES_OVERWRITE: true
+        CMA_HEALTH_RULES_ONLY: true
+      run: |
+          echo Running health rules only, get values from environment variables,
+          ./start.sh -a Jenkins_API -c configmyappdemo-2044no-uzyczrm0.appd-cx.com -p appd -u appd
+
+    - name: TestCase9- health rules only - overwrite default - config
+      env:
+        CMA_USE_HTTPS: false
+      run: |
+          cp config.json config.json.bkp
+          curl https://gist.githubusercontent.com/AlexJov/63ccb17421208679ef63b55afafea712/raw/b8e5ebc5399a8d7df5422ff07c49c892f0c3bd63/config.json -o ./config.json
+          echo Running health rules only, get values from config,
+          ./start.sh -a Jenkins_API -c configmyappdemo-2044no-uzyczrm0.appd-cx.com -p appd -u appd
+          cp config.json.bkp config.json
+
+    - name: TestCase10- delete health rules, existing,
+      env:
+        CMA_USE_HTTPS: false
+      run: |
+          curl https://gist.githubusercontent.com/AlexJov/d65f5e436bd54dd53aaad679f6b8d8e9/raw/57f3d2ec115f7411944b934c735f57a6a276d44b/Agent%2520Availability%2520to%2520Delete.json -o ./health_rules/Application/AgentAvailabilityToDelete.json
+          curl https://gist.githubusercontent.com/AlexJov/03317fd4271325fbd6678dded2df6e91/raw/bb33a4b3abcaed762f1a5b262586183c6efd4402/CpuUtilisationTooHighToDelete.json -o ./health_rules/ServerVisibility/CpuUtilisationTooHighToDelete.json
+          
+          echo Running health rules only, import additional health rules,
+          ./start.sh -a Jenkins_API -c configmyappdemo-2044no-uzyczrm0.appd-cx.com -p appd -u appd --include-sim --health-rules-only
+
+          echo Delete health rules:
+          ./start.sh -c http://configmyappdemo-2044no-uzyczrm0.appd-cx.com/ -a Jenkins_API -u appd -p appd --health-rules-delete "Agent Availability to Delete, Server Health: CPU Utilisation is too high to Delete"
```

Comment:
Still, here it shows like the Health Rule does not exist although I imported it in the previous step (line 101) :/